binder/environment.yml

```diff
@@ -79,15 +79,17 @@ dependencies:
   - pamela=1.0.0=py_0
   - pandocfilters=1.5.0=pyhd8ed1ab_0
   - parso=0.8.3=pyhd8ed1ab_0
-  - pickleshare=0.7.5=py_1003
+  - pathspec=0.9.0=pyhd8ed1ab_0
+  - platformdirs=2.3.0=pyhd8ed1ab_0
   - poyo=0.5.0=py_0
   - prometheus_client=0.12.0=pyhd8ed1ab_0
   - ptyprocess=0.7.0=pyhd3deb0d_0
+  - pure_eval=0.2.1=pyhd8ed1ab_0
   - pycparser=2.21=pyhd8ed1ab_0
   - pyjwt=2.3.0=pyhd8ed1ab_1
   - pyparsing=3.0.6=pyhd8ed1ab_0
   - python-json-logger=2.0.1=pyh9f0ad1d_0
-  - python_abi=3.10=2_cp310
+  - python_abi=3.9=2_cp39
```

Comment:
Looks like you have Python 3.9 in your local environment versus mine with 3.10 when I recently created this lock file. We need to figure out how we all stay within the same major Python versions. Otherwise there will be a lot of back and forth when adding/updating packages.
Or we should update the script that updates this file to create an environment based of the previous lock file, then add the package and create a new lock file.
Will log an issue

Reply:
#63